apps/next-evans/pages/static/[static].tsx

```diff
@@ -0,0 +1,136 @@
+import matter from 'gray-matter';
+import cx from 'classnames';
+import { MDXRemote } from 'next-mdx-remote';
+import { serialize } from 'next-mdx-remote/serialize';
+import invariant from 'tiny-invariant';
+import { type MDXRemoteSerializeResult } from 'next-mdx-remote';
+import type { GetStaticPaths, GetStaticProps } from 'next';
+import type { MdxPageFrontmatter } from '@/lib/types';
+import { WhiteOverlay } from '../explore';
+import HeroImg from '@/public/images/explore-books.jpg';
+import * as mdx from '@/lib/mdx';
+import { LANG } from '@/lib/env';
+import BackgroundImage from '@/components/core/BackgroundImage';
+import { getAllDocuments } from '@/lib/db/documents';
+
+export const getStaticPaths: GetStaticPaths = async () => ({
+  paths: mdx
+    .fileData()
+    .filter((file) => file.lang === LANG)
+    .map(({ slug }) => ({ params: { static: slug } })),
+  fallback: false,
+});
+
+export const getStaticProps: GetStaticProps<Props> = async (context) => {
+  const slug = context.params?.static;
+  invariant(typeof slug === `string`);
+
+  const englishBooks = Object.values(await getAllDocuments(`en`));
+  const spanishBooks = Object.values(await getAllDocuments(`es`));
+  const numAudiobooks =
+    LANG === `en`
+      ? englishBooks.filter((book) => book.hasAudio).length
+      : spanishBooks.filter((book) => book.hasAudio).length;
+
+  const source = mdx
```

Comment:
i'm afraid we'll forget this later, so, even though we're not currently leveraging the description for SEO, can you extract out a function that does this replacing, and then also run the `frontmatter.description` through it as well? down on line 43.5...